dev-support/hbase_nightly_yetus.sh

```diff
@@ -40,6 +40,10 @@ if [[ -n "${MULTIJDK}" ]]; then
   YETUS_ARGS=("--multijdkdirs=${MULTIJDK}" "${YETUS_ARGS[@]}")
 fi
 
+# If we're doing docker, make sure we don't accidentally pollute the image with a host java path
```

Comment:
Does this ever not run in Docker mode? Can we settle on only running in Docker mode? It keeps things simpler and tidy, IMHO. The possibility of host environment values leaking in makes for bizarre bugs.

Reply:
AFAICT only our general precommit job can run not in docker mode. We've only needed that for debugging a few times.

It would be nice if the nightly bootstrap script didn't require docker because running in docker on OSX is prohibitively slow.

Reply:
So we do run nightly on not-Jenkins? Or you mean for developers doing local testing...

What's the issue with it running slowly on OSX? Is that an upstream bug tracked somewhere?